src/Xamarin.Android.Build.Tasks/Utilities/MarshalMethodsAssemblyRewriter.cs

```diff
@@ -114,47 +114,37 @@ public void Rewrite (DirectoryAssemblyResolver resolver, List<string> targetAsse
 				}
 			}
 
-			var newAssemblyPaths = new List<string> ();
+			var newAssemblyPaths = new List<ValueTuple<string, string>> ();
```

Comment:
Can we not use the glorious `(string Something, string SomethingElse)` tuple syntax?